examples/getting-started.md

```diff
@@ -0,0 +1,135 @@
+# Kubeplus
+
+## Getting Started with an example
+
+Let’s look at an example of creating a multi-instance WordPress Service using KubePlus. The WordPress service provider goes through the following steps towards this on their cluster:
+
+1. Create cluster or use an existing cluster. For testing purposes you can create a [minikube](https://minikube.sigs.k8s.io/docs/) or [kind](https://kind.sigs.k8s.io/) cluster:
```

Comment:
Move steps 1, 2, 3 under manual install section in the README.

Move step 4 under scripted install section in the README.

Under Getting started section, put only a one line note: "If you have not setup KubePlus, follows the steps in the README to setup KubePlus first."

Reply:
Updated to inculcate the changes.

Reply:
Updated the README to inculcate the changes.